README.md

```diff
@@ -57,30 +63,20 @@ For older versions, see [Tableau Connector SDK legacy documentation].
 
 ## Usage
 
-### Configure data source
-
-Todo: Add a few words how to actually establish the connection to CrateDB
-after installing the driver and connector?
-
-
-### Signature verification
+For development purposes, please read the [sandbox documentation](./DEVELOP.md).
 
-The current release of the connector is not signed. This issue is being addressed,
-but in the meantime, signature verification can be disabled.
-
-* On _Tableau Desktop_, please use this command-line argument: `-DDisableVerifyConnectorPluginSignature=true`
-
-* On _Tableau Server_, you can disable signature verification by setting `native_api.disable_verify_connector_plugin_signature` to `true` via TSM.
+### Configure data source
 
-More information can be found on the [Tableau Connector SDK `run-taco` documentation].
+When it comes to configuring a data source in Tableau, a few fragments in the
+[sandbox documentation](./DEVELOP.md) may also be applicable to a production setup.
 
+Todo: Add a dedicated section about how to install and configure the connector
+in non-development mode.
```

Comment:
Just another side note: This bears a little backlog item, where we could make the docs more concise, probably by repurposing fragments of `DEVELOP.md`? I'll be happy to jot that down, if you can give me a quick 5 minute in-person  orientation flight?

Reply:
what do you mean in non-development mode